connectors/sources/atlassian.py

```diff
@@ -17,6 +17,11 @@
 RETRIES = 3
 RETRY_INTERVAL = 2
 USER_BATCH = 50
+DATA_CENTER_USER_BATCH = 1000
```

Comment:
What does `DATA_CENTER_USER_BATCH` specify exactly? Is this the max amount of users we can fetch in one request? Is it the same value as in jira.py? https://github.com/elastic/connectors/blob/1d4fb789f5588d018907aef786b055ba605dc562/connectors/sources/jira.py#L43

If they're the same value, I suggest having it defined in this file (atlassian.py) and importing it into any sources that use it.

Reply:
I'm also confused about `DATA_CENTER_USER_BATCH`, specifically because below it seems to be not for DataCenter, but for anything non-cloud:

```
                f"{url}?startAt={start_at}"
                if self.source.configuration["data_source"]
                in [JIRA_CLOUD, CONFLUENCE_CLOUD]
                else url.format(start_at=start_at, max_results=DATA_CENTER_USER_BATCH)
```

Reply:
> If they're the same value, I suggest having it defined in this file (atlassian.py) and importing it into any sources that use it.

The value is definitely same but the purpose is different at https://github.com/elastic/connectors/blob/1d4fb789f5588d018907aef786b055ba605dc562/connectors/sources/jira.py#L43 and it is used for both cloud and non-cloud data source. where in `atlassian.py`, the value is used for fetching just non-cloud users.

> I'm also confused about DATA_CENTER_USER_BATCH, specifically because below it seems to be not for DataCenter, but for anything non-cloud:

renaming the constant to get more clear idea.